app/views/comments/_comment.html.erb

```diff
@@ -8,11 +8,8 @@
   <%= comment.body %>
 </p>
 
-
-<% if current_user && current_user.editor? %> 

```

Comment:
do przywrocenia. masz jeszcze to `editor?` w ogole? chyba juz to usuwales z usera?